l1-contracts/contracts/bridgehub/MessageRoot.sol

```diff
@@ -154,46 +154,4 @@ contract MessageRoot is IMessageRoot, ReentrancyGuard {
         // slither-disable-next-line unused-return
         sharedTree.updateAllLeaves(newLeaves);
     }
-
-    // It is expected that the root is present
-    // `_updateTree` should be false only if the caller ensures that it is followed by updating the entire tree.
-    function _unsafeResetChainRoot(uint256 _index, bool _updateTree) internal {
```

Comment:
This is deleted because it is not used for now, right?

Reply:
yup and the future of the calling method is in the air